### PR TITLE
Minor edits in files

### PR DESCRIPTION
**Pull request by cduangprom**
Is there a comma needed in these lines? 
* @dev emits ExitOmitted event if the exit is omitted.
So it will look like this:  @dev emits ExitOmitted event, if the exit is omitted.  ADD A COMMA?
Also
ExitFinalized event if the exit is processed and funds are withdrawn
So it will look like this: 
ExitFinalized event, if the exit is processed and funds are withdrawn

This: @param exitMap The storage of all standard exit data.
Or this: @param exitMap Stores all standard exit data.

### CODE REVIEW

File: plasma_framework/contracts/src/exits/payment/spendingConditions/PaymentOutputToPaymentTxCondition.sol [hunk 1]
```diff
@@ -51,19 +51,19 @@ contract PaymentOutputToPaymentTxCondition is ISpendingCondition {
...
         UtxoPosLib.UtxoPos memory utxoPos = UtxoPosLib.build(TxPosLib.TxPos(inputTxPos), outputIndex);
         require(
             spendingTx.inputs[inputIndex] == bytes32(utxoPos.value),
-            "The spending tx does not point to the correct utxo position of the output"
+            "The spending tx is pointing to the incorrect output UTXO position"
         );
 
         address payable owner = inputTx.outputs[outputIndex].owner();
-        require(owner == ECDSA.recover(eip712.hashTx(spendingTx), signature), "Tx not correctly signed");
+        require(owner == ECDSA.recover(eip712.hashTx(spendingTx), signature), "Tx signature is incorrectly signed");
```
Comment (thec00n) [inline plasma_framework/contracts/src/exits/payment/spendingConditions/PaymentOutputToPaymentTxCondition.sol]: Tx is incorrectly signed

File: plasma_framework/contracts/src/utils/OnlyFromAddress.sol [hunk 2]
```diff
@@ -3,7 +3,7 @@ pragma solidity 0.5.11;
 contract OnlyFromAddress {
 
     modifier onlyFrom(address caller) {
-        require(msg.sender == caller, "Not being called by expected caller");
+        require(msg.sender == caller, "Caller address is invalid");
```
Comment (thec00n) [inline plasma_framework/contracts/src/utils/OnlyFromAddress.sol]: `require(msg.sender == caller, "Caller address is not authorized");`

File: plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouterArgs.sol [hunk 3]
```diff
@@ -16,17 +16,17 @@ library PaymentStandardExitRouterArgs {
...
-     * @param inputIndex Which input of the challenging tx corresponds to the exiting output.
-     * @param witness Witness data that can prove the exiting output is spent.
-     * @param spendingConditionOptionalArgs optional extra data for the spending condition.
-     * @param outputGuardPreimage (Optional) output guard preimage for the challenge tx to use the output
-     * @param challengeTxPos (Optional) tx position of the challenge tx if it is of MVP protocol.
-     * @param challengeTxInclusionProof (Optional) if the challenge tx is of MVP protocol provide the inclusion proof of it
-     * @param challengeTxConfirmSig (Optional) if the challenge tx is of MVP protocol provide the confirm sig of it
+     * @notice Input args data for challengeStandardExit
+     * @param exitId Identifier of the standard exit to challenge
+     * @param exitingTx The exiting transaction
```
Comment (pdobacz) [inline plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouterArgs.sol]: let's be consistent with the next line:

`RLP-encoded transaction that creates the exiting output`

File: plasma_framework/contracts/src/utils/RLP.sol [hunk 4]
```diff
@@ -55,9 +55,9 @@ library RLP {
     }
 
     /**
-     * @dev Creates an RLPItem from an array of RLP encoded bytes.
-     * @param self The RLP encoded bytes.
-     * @param strict Will throw if the data is not RLP encoded.
+     * @dev Creates an RLPItem from an array of RLP-encoded bytes
+     * @param self The RLP-encoded bytes
+     * @param strict Triggered when data is not RLP-encoded
```
Comment (pdobacz) [inline plasma_framework/contracts/src/utils/RLP.sol]: not sure but it seems both old and new version is misleading. Shouldn't this be:

`If True, will throw if the data is not RLP-encoded`

But this is just an educated guess, can someone confirm?

File: plasma_framework/contracts/src/vaults/Vault.sol [hunk 5]
```diff
@@ -6,17 +6,17 @@ import "../framework/utils/Operated.sol";
...
 contract Vault is Operated {
     event SetDepositVerifierCalled(address nextDepositVerifier);
     PlasmaFramework internal framework;
     bytes32[16] internal zeroHashes; // Pre-computes zero hashes to be used for building merkle tree for deposit block
 
     /**
-     * @notice Stores deposit verifier contracts addresses where first was effective upto
-     *  `newDepositVerifierMaturityTimestamp` point of time and second become effective after
+     * @notice Stores deposit verifier contract addresses, where the first contract address is effective until the 
+     *  `newDepositVerifierMaturityTimestamp`, after which the second contract address becomes effective
```
Comment (pdobacz) [inline plasma_framework/contracts/src/vaults/Vault.sol]: I'm guessing this is a moment in time, not an interval, so `after which` doesn't ift.

File: plasma_framework/contracts/src/vaults/verifiers/Erc20DepositVerifier.sol [hunk 6]
```diff
@@ -7,19 +7,19 @@ import {PaymentTransactionModel as DepositTx} from "../../transactions/PaymentTr
...
+    // Hardcoded transaction type for payment transaction
     uint8 constant private DEPOSIT_TX_TYPE = 1;
 
     /**
      * @notice Overrides the function of IErc20DepositVerifier and implements the verification logic
-     *         for Payment transaction
-     * @dev ERC20 token must be approved to the vault address beforehand
-     * @return verified (owner, token, amount) of the deposit ERC20 token data
+     *         for payment transaction
+     * @dev ERC20 token requires pre-approval to the vault address
```
Comment (pdobacz) [inline plasma_framework/contracts/src/vaults/verifiers/Erc20DepositVerifier.sol]: `Vault address must be approved to transfer from the depositor's address before doing the deposit` or similr

File: plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol [hunk 7]
```diff
@@ -4,10 +4,10 @@ pragma experimental ABIEncoderV2;
 interface IStateTransitionVerifier {
 
     /**
-    * @notice Verifies state transition logic.
-    * @param txBytes the targeting transaction that is doing the state transition verification
-    * @param inputTxs input transactions of the targeting transaction checking state transition
-    * @param outputIndexOfInputTxs the output index of the input txs that the transaction input is pointing to
+    * @notice Verifies state transition logic
+    * @param txBytes The targeting transaction that verifies the state transition
```
[the remark this answers is not in the document]
Comment (boolafish) [reply plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol]: yeah, I second this. I think it was mis-interpreted by my bad English on previous comment ><

[the remark this answers is not in the document]
Comment (cduangprom) [reply plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol]: Not clear to me at all what is meant by this line: The transaction that does the state transition   to verify. ('to verify' at the end is confusing).

File: plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol [hunk 8]
```diff
@@ -4,10 +4,10 @@ pragma experimental ABIEncoderV2;
...
+    * @param inputTxs Input transactions of the targeting transaction that verifies the state transition
```
[the remark this answers is not in the document]
Comment (cduangprom) [reply plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol]: Also not clear: "Input transaction to the transaction to verify"
Implication is: This is an input transaction ...  to another transaction .. and that other transaction must be verified. 
Is that what this means?

File: plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol (hunk 7 above)
[the remark this answers is not in the document]
Comment (boolafish) [reply plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol]: > Is the intended meaning: This transaction does the state transition. This transaction must be verified.

pretty much. though for me the first part is more like "This transaction has the state transition"? (I feel the transaction itself does not do any action but log and hold the transition information.)

File: plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol (hunk 8 above)
[the remark this answers is not in the document]
Comment (boolafish) [reply plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol]: yes. The "another transaction" is the one on `txBytes`

**Comment by boolafish** [comment]
this has been done in another PR. closing this.

**Pull request closed without merging**